pages/Category.vue

```diff
@@ -604,127 +613,123 @@ export default {
     margin: 0 auto;
   }
 }
-.main {
-  &.section {
-    padding: var(--spacer-xs);
-    @include for-desktop {
-      padding: 0;
-    }
-  }
-}
+
 .breadcrumbs {
   padding: var(--spacer-base) var(--spacer-base) var(--spacer-base) var(--spacer-sm);
 }
+
+.content {
+  display: grid;
+  grid: "navbar" "products";
+
+  @include for-desktop{
+    grid: "title      navbar"
+          "categories products"
+          / 15% 1fr;
+  }
+}
+
+.title {
+    grid-area: title;
+    padding: var(--spacer-sm);
+    border: 1px solid var(--c-light);
+    border-width: 1px 1px 1px 0;
+  }
+
 .navbar {
-  position: relative;
-  display: flex;
+  grid-area: navbar;
+  display: grid;
+  grid:"filter counter sort" / 1fr minmax( auto, max-content) 1fr;
```

Comment:
I think a comment here that describes how to expand this grid during customization in plain english would be helpful. Something like "To add more items to the navbar, add their grid-area to the template. The mobile navbar shouldn't have more than 3 items at a time, so replace rather than add."